package.json

```diff
@@ -3,5 +3,10 @@
   "version": "1.0.0",
   "main": "index.js",
   "repository": {},
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
+    "electron": "^1.6.11",
```

Comment:
I think I had added electron for testing something with nightmare, I've removed it